yabt/builders/custom_installer.py

```diff
@@ -156,9 +160,11 @@ def archive_handler(unused_build_context, target, fetch, package_dir, tar):
     if ext in ('.gz', '.bz2', '.tgz'):
         with tarfile.open(package_dest, 'r:*') as src_tar:
             src_tar.extractall(extract_dir)
-    elif ext in ('.zip',):
-        with ZipFile(package_dest, 'r') as zipf:
-            zipf.extractall(extract_dir)
+    # TODO(shai) https://stackoverflow.com/questions/39296101/python-zipfile-removes-execute-permissions-from-binaries
+    # We deprecated this features since it's broken
+    # elif ext in ('.zip',):
```

Comment:
can we make this deprecation more user-visible?

e.g., keep the `elif`, and raise a deprecation warning exception so the user will see an error message if they have `zip` archives